go.mod

```diff
@@ -2,12 +2,15 @@ module github.com/envoyproxy/gateway
 
 go 1.22.4
 
+replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.16
+
 require (
 	fortio.org/fortio v1.65.0
 	fortio.org/log v1.12.2
 	github.com/Masterminds/semver/v3 v3.2.1
 	github.com/cncf/xds/go v0.0.0-20240423153145-555b57ec207b
-	github.com/davecgh/go-spew v1.1.1
+	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
+	github.com/docker/cli v26.1.3+incompatible
```

Comment:
any way to avoid the docker dependency and use more generic oci libs here?

Reply:
The docker lib is used to parse docker compatible config file to get auth info.